script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2.txt

```diff
@@ -205,7 +205,7 @@ If he is absent, today's class will be self-study. The rest of the time can be u
 Some students use it to study, some to play.
 <Page58>
 I start to work on my math problems.
-When I was a child, I was told that it would be difficult for me to enter a prestigious school with my body, so I stopped going to cram school in middle school, but studying itself was still interesting.
+When I was a child, I was told that it would be difficult for me to enter a prestigious school with my body, so I stopped going to cram school in middle school, but studying itself was still interesting. //School repeated 3 times?
```

Comment:
Sometimes it's just hard to avoid =/

Reply:
are these spaces added manually when commenting? these spaces show up in game

Reply:
Ooh, good point. Generally manual editors of the files will add the space for readability, so there's a lot of instances:
```
ross@mjolnir:/h/r/g/T/s/1_0_2current_en_script$ gg ' //' | wc -l
53
```
We can just fix this handling in deepLuna.

Reply:
it doesnt bother me personally.